test/test_torch.py

```diff
@@ -6530,7 +6531,10 @@ def import_module(name, filename):
             return module
 
         with filecontext_lambda() as checkpoint:
-            fname = os.path.join(os.path.dirname(__file__), 'data/network1.py')
+            try:
+                fname = get_file_path('test', 'data', 'network1.py')
+            except IOError:
+                fname = get_file_path('data', 'data', 'network1.pyc')
```

Comment:
Explain with a comment why the fallback on pyc is desirable.

Reply:
don't know, but this is existing behavior

Reply:
this change can be removed, we have to add `compile=False` to the unittest TARGETS. I recently got to know that.